src/App.tsx

```diff
@@ -15,48 +16,111 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type State = {
+  isReversed: boolean,
+  sortType: SortType,
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: State,
+) {
+  const visibleGoods = [...goods];
+
+  visibleGoods.sort((g1, g2) => {
```

Comment:
> `g1`, `g2`

How much do your pay for key action? 😄

Reply:
that was a joke, I just meant that it is not good thing to name your variables with letters. We want our code to be maximally clear. So better to come up with sth like `currentGood`, `nextGood`